### PR TITLE
[DOCFIX] Add a section about Active Sync for HDFS

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@calvinjia PTAL

File: docs/en/advanced/Namespace-Management.md [hunk 1]
```diff
@@ -286,6 +286,43 @@ $ ls /tmp/alluxio-demo
...
+#### Quiet period for Active Sync
+
+Active Sync also tries to avoid synchronization when the target directory is being heavily used.
+It tries to look for a quiet period in UFS activity to start syncing between the UFS and the Alluxio space, to avoid overloading the UFS when it is busy.
+There are two configuration options that control this behavior.
+
+`alluxio.master.activesync.maxactivity` is the maximum number of changes in the UFS directory to be considered "quiet".
+`alluxio.master.activesync.maxage` is the maximum number of intervals we will wait before synchronizing the UFS and the Alluxio space.
+
+The system guarantees that we will start syncing a directory if it is "quiet", or it has not been synced for a long period (period longer than the max age).
```
Comment (calvinjia) [inline docs/en/advanced/Namespace-Management.md]: Could you provide an example of these parameters being used in hypothetical situations?

File: docs/en/advanced/Namespace-Management.md [hunk 2]
```diff
@@ -286,6 +286,43 @@ $ ls /tmp/alluxio-demo
 hello
 ```
 
+### Metadata Active Sync for HDFS
+In version 2.0, we introduced a new feature for maintaining synchronization between Alluxio space and the UFS when the UFS is HDFS.
+The feature, called active sync, listens for HDFS events and periodically synchronizes the metadata between the UFS and Alluxio namespace as a background task.
+
+To designate a directory to be actively synchonized, issue the following Alluxio command.
```
[the remark this answers is not in the document]
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: done

File: docs/en/advanced/Namespace-Management.md (hunk 1 above)
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: done

File: docs/en/advanced/Namespace-Management.md [hunk 3]
```diff
@@ -286,6 +286,43 @@ $ ls /tmp/alluxio-demo
 hello
 ```
 
+### Metadata Active Sync for HDFS
+In version 2.0, we introduced a new feature for maintaining synchronization between Alluxio space and the UFS when the UFS is HDFS.
+The feature, called active sync, listens for HDFS events and periodically synchronizes the metadata between the UFS and Alluxio namespace as a background task.
```
[the remark this answers is not in the document]
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: done

**Comment by yuzhu** [comment]
@calvinjia @apc999 comments addressed, PTAL.

File: docs/en/advanced/Namespace-Management.md [hunk 4]
```diff
@@ -286,6 +286,56 @@ $ ls /tmp/alluxio-demo
...
+
+The system guarantees that we will start syncing a directory if it is "quiet", or it has not been synced for a long period (period longer than the max age).
+
+For example, the following setting 
+
+```
+alluxio.master.activesync.interval=30secs
+alluxio.master.activesync.maxactivity=100
+alluxio.master.activesync.maxage=5
+
```
Comment (apc999) [inline docs/en/advanced/Namespace-Management.md]: Extra newline

File: docs/en/advanced/Namespace-Management.md [hunk 5]
```diff
@@ -286,6 +286,56 @@ $ ls /tmp/alluxio-demo
 hello
 ```
 
+### Metadata Active Sync for HDFS
+In version 2.0, we introduced a new feature for maintaining synchronization between Alluxio space and the UFS when the UFS is HDFS.
+The feature, called active sync, listens for HDFS events and periodically synchronizes the metadata between the UFS and Alluxio namespace as a background task on the master.
+
+To designate a directory for active syncing, issue the following Alluxio command.
```
Comment (calvinjia) [inline docs/en/advanced/Namespace-Management.md]: Could we specifically stick to the phrase `active sync` and use it only as a noun? For example `To enable active sync on a directory ...`

Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: done

**Comment by yuzhu** [comment]
@apc999 @calvinjia comments addressed, PTAL

File: docs/en/advanced/Namespace-Management.md (hunk 4 above)
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: fixed

**Comment by yuzhu** [comment]
jenkins, test this please

**Comment by yuzhu** [comment]
@calvinjia could you please merge this doc?

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**